lib/ansible_spec/load_ansible.rb

```diff
@@ -268,6 +268,26 @@ def self.name_exist?(array)
     end
   end
 
+  # param: hash
+  # param: variable file
+  def self.load_vars_file(vars, path)
+    vars_file = path
+    if File.exist?(vars_file)
+      yaml = YAML.load_file(vars_file)
+      if yaml.kind_of?(Hash)
+        vars.merge!(yaml)
+      end
+    else
+      vars_file = path+".yml"
+      if File.exist?(vars_file)
+        yaml = YAML.load_file(vars_file)
+        if yaml.kind_of?(Hash)
+          vars.merge!(yaml)
+        end
+      end
+    end
```

Comment:
If there is no problem, Could you add `return vars` at end of load_vars_file?
Because (I think) the function should respond return of value for readability and testability.